assets/scripts/script.js

```diff
@@ -73,20 +73,41 @@
     }
 
     // Adds lightbox functionality to product gallery.
-    if (typeof $.fn.lightGallery === 'function') {
-      $('.js-gallerya-lightbox').lightGallery({
-        thumbnail: true,
-        showThumbByDefault: false,
-        subHtmlSelectorRelative: true,
-        selector: '.gallerya__image > a'
-      });
-      $('.woocommerce-product-gallery').lightGallery({
-        thumbnail: true,
-        showThumbByDefault: false,
-        subHtmlSelectorRelative: true,
-        selector: '.woocommerce-product-gallery__image > a'
-      });
-    }
+    $().fancybox({
+      selector : '.js-gallerya-lightbox .gallerya__image > a, .woocommerce-product-gallery .woocommerce-product-gallery__image > a',
+      closeExisting: false,
+      loop: true,
+      keyboard: true,
+      buttons: [
+        'zoom',
+        'fullScreen',
+        'share',
+        'download',
+        'thumbs',
+        'close'
+      ],
+      animationEffect: 'zoom',
+      animationDuration: 250,
+      transitionEffect: 'slide',
+      thumbs: {
+        autoStart: false,
+        axis: 'x'
+      },
+      lang: fancyboxTranslations.language,
+      de: {
+        CLOSE: "Schliessen",
+        NEXT: "Weiter",
+        PREV: "Zurück",
+        ERROR: "Die angeforderten Daten konnten nicht geladen werden. <br/> Bitte versuchen Sie es später nochmal.",
+        PLAY_START: "Diaschau starten",
+        PLAY_STOP: "Diaschau beenden",
+        FULL_SCREEN: "Vollbild",
+        THUMBS: "Vorschaubilder",
+        DOWNLOAD: "Herunterladen",
+        SHARE: "Teilen",
+        ZOOM: "Maßstab"
+      }
```

Comment:
I'd consider to move these translations to PHP and pass them with [`wp_localize_script`](https://developer.wordpress.org/reference/functions/wp_localize_script/). In that way we could easily control translations with PoEdit.